src/app/components/InfoGraphicSection.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from "@nextui-org/react";
+import { Icon } from "@phosphor-icons/react";
+import Image, { StaticImageData } from "next/image";
+import Link from "next/link";
+
+interface InfoGraphicBase {
+  header: string | JSX.Element;
+  body: string;
+  link?: {
+    icon: Icon;
+    label: string;
+    href: string;
+  };
+}
+
+interface InfoGraphicWithImage extends InfoGraphicBase {
+  image: {
+    data: StaticImageData;
+    alt: string;
+    className?: string;
+  };
+}
+
+interface InfoGraphicWithComponent extends InfoGraphicBase {
+  component: JSX.Element;
+}
+
+type InfoGraphicProps = InfoGraphicWithImage | InfoGraphicWithComponent;
+
+/**
+ * Renders an info graphic section with a header, body text, and optional link and graphic content,
+ * depending on the usage of the 'image' or 'component' prop.
+ *
+ * Initially developed for use on the 'pitch deck' landing page.
+ */
+export const InfoGraphicSection = (props: InfoGraphicProps) => {
+  const { header, body, link } = props;
+
+  // Dynamically renders the graphic content based on prop type.
+  const renderGraphicContent = () => {
+    if ("image" in props) {
+      return (
+        <Image
```

Comment:
@brandonfcohen1 I had question regarding this repos usage of images: I saw that some existing images were stored in the `/public` folder, where its path was directly reference as a string from `import { Image } from "@nextui-org/react"`. What were the choices made to not use `Image from next/image;` instead?

As far as I'm aware, by using `next/image`, you get optimized image compressions and images are sized and served based on the users viewports that I don't believe you'd get by just hosting directly from the `/public` folder.

Reply:
Actually yea, I think my hunch is correct — running lighthouse on the prod homepage gives 86 for the desktop and 68 for mobile, with big penalties coming from the unoptimized images. Might be worth creating a ticket to refactor all usages of `Image from @next-ui` to `Image from next/image`

desktop
![desktop](https://github.com/CodeForPhilly/vacant-lots-proj/assets/8061917/af281510-db1e-4447-b062-83dea6564094)

mobile
![mobile](https://github.com/CodeForPhilly/vacant-lots-proj/assets/8061917/66c7962c-b43c-4616-86db-7a670b7ed5c0)

Reply:
@paulhchoi lol the choice is 100% just me learning React on the fly + relying heavily on chatgpt + copilot. Change what you want. :)